app/src/main/java/com/github/studydistractor/sdp/MainActivity.kt

```diff
@@ -1,13 +1,29 @@
 package com.github.studydistractor.sdp
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import androidx.ui.core.Text
-import androidx.ui.core.setContent
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.activity.compose.setContent
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.ktx.database
+import com.google.firebase.ktx.Firebase
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContent{Text("Hello World !")}
+        setContent{
+            Column {
```

Comment:
It would be nice to center the content here by 
```
 Column(
                modifier = Modifier.fillMaxWidth().fillMaxHeight(),
                verticalArrangement = Arrangement.Center,
                horizontalAlignment = Alignment.CenterHorizontally
            ) { ...
```